Loop54.Test.AspNetCore/Views/GetRelatedEntities/Index.cshtml

```diff
@@ -5,10 +5,13 @@
 }
 <form id="GetRelatedEntitiesForm" method="POST">
     <div>
-        <label for="QueryText">Id</label>
+        <label for="IdText">Id</label>
         <input type="text" id="IdText" name="id" autofocus="" value="13" />
-        <label for="QueryText">Type</label>
-        <input type="text" id="TypeText" name="type" autofocus="" value="Product" />
+        <label for="TypeText">Type</label>
+        <input type="text" id="TypeText" name="type" autofocus="" value="Product" /><br/>
+        <label>Relation kind</label><br/>
+        <input type="radio" id="similarRadio" name="relationKind" value="similar" checked/><label for="similarRadio">similar</label><br />
```

Comment:
This will fail until `helloworld` endpoint is upgraded to support requests with RelationKind.

Reply:
It's also valid to not specify this, though, so I would prefer a dropdown with an empty option. That would also allow it to continue working with the current helloworld engine, which may not be upgraded for quite some time.

Reply:
Reworked that to use dropdown with `(no value)` option